mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -1,137 +1,293 @@
+'use strict';
+
 import React from 'react'
 import {reduxForm} from 'redux-form';
-import { Modal } from "react-bootstrap";
+import Modal from 'react-modal';
```

Comment:
It is a popular library for showing and controlling modals, I try to use it everywhere where modals exist. We could remove it but then I would have to recreate the functionality that it has such as show hide from props. 

We are going to use modals a lot so it is good if we have a standard way of using them